tests/web_client_specs/containerViewSpec.js

```diff
@@ -0,0 +1,159 @@
+/* globals girderTest, describe, it, expect, waitsFor, runs */
+girderTest.importPlugin('jobs', 'worker', 'slicer_cli_web');
+
+var slicer;
+girderTest.promise.done(function () {
+    slicer = girder.plugins.slicer_cli_web;
+});
+
+girderTest.startApp();
+
+function login(user, password) {
+    girderTest.waitForLoad('login wait 1');
+    runs(function () {
+        $('.g-login').click();
+    });
+
+    girderTest.waitForDialog('login wait 2');
+    runs(function () {
+        $('#g-login').val(user || 'user');
+        $('#g-password').val(password || 'password');
+        $('#g-login-button').click();
+    });
+
+    waitsFor(function () {
+        return $('.g-user-dropdown-link').length > 0;
+    }, 'user to be logged in');
+    girderTest.waitForLoad('login wait 3');
+}
+
```

Comment:
I'm not sure why the dockerTaskSpec is written this way, but we should be able to just use `girderTest.login` instead of having a function defined here.